pkg/sbom/cpe.go

```diff
@@ -47,6 +47,13 @@ func generateWfnAttributesForAPK(p pkgInfo) *wfn.Attributes {
 		return &attr
 	}
 
+	if strings.HasPrefix(name, "gitlab-") {
```

Comment:
should we add a comment here in the code to explain why that?

Reply:
maybe the one you wrote in the pr summary

```
Syft has been generating lots of CPEs for GitLab components, but they're never correct.
```

Reply:
it is just a suggestion not a blocker for this pr

Reply:
Okay cool — this whole file (`cpe.go`) exists to tackle this problem, FWIW. This is just the case where it affects our GitLab matching 😢